test/index.html

```diff
@@ -13,9 +13,12 @@
     <script type="module" src="./lib/unsafe-html_test.js"></script>
     <script type="module" src="./lib/async-append_test.js"></script>
     <script type="module" src="./lib/async-replace_test.js"></script>
+    <script type="module" src="./lib/modify-template_test.js"></script>
     <script>
       WCT.loadSuites([
         'shady.html',
+        'shady.html?shadydom=true',
```

Comment:
I keep forgetting about this... I wonder if we should do this for all the tests?

Reply:
let's revisit later.